modules/developer_manual/pages/core/acceptance-tests.adoc

```diff
@@ -254,8 +254,8 @@ Run a command like the following:
 
 [source,bash]
 ----
-sudo -u www-data make test-acceptance-api BEHAT_FEATURE=tests/acceptance/features/apiTags/createTags.feature
-sudo -u www-data make test-acceptance-cli BEHAT_FEATURE=tests/acceptance/features/cliProvisioning/addUser.feature
+make test-acceptance-api BEHAT_FEATURE=tests/acceptance/features/apiTags/createTags.feature
```

Comment:
Is this docs only for running on drone? Because I need to add `TEST_SERVER_URL` in the docs for tests to run. And, doing this works on every setups, doesn't it? 

EDIT: Pressed the wrong button

Reply:
That should be described in sections above. There are about 100 ways to do it.
For example, when I open a terminal in the morning, I do:
```
export TEST_SERVER_URL=http://172.17.0.1:8080
export TEST_SERVER_FED_URL=http://172.17.0.1:8180
export BROWSER=chrome
```
and then I do not have to think about putting any of these before or after ``make`` commands.

So the "style" of using environment variables it is up to the developer. The documentation presents one "style".